docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -20,6 +20,7 @@ labels:
     - oss
 menuTitle: Configure data links
 title: Configure data links
+description: Create data links and data link variables to create links between panels
```

Comment:
minor nit: 
```suggestion
description: Configure data links to create links between dashboards and link to external resources
```

Maybe we can also include info on that data links can be configured with template variables / panel data embedded but 1 sentence description might not be sufficient to also express this 😬

Reply:
Yeah these are meant to be pretty concise, but I'll make the edit you've suggested